README.rst

```diff
@@ -14,50 +14,57 @@ What it is
 
 - remove need of urls.py in favor of nested views,
 - nested security checking support,
-- take DRY to another level with nested views.
+- takes DRY to another level with view route parenting,
 - django can you generate a menu is: yes,
 - modern pattern for creating your own admin-like CRUD,
 
 Extracted from CRUDLFA+, re DDD'd, under TDD.
 
+Target:
+
 .. code-block:: python
 
     # yourapp.views
-    urlpatterns = xcbv.View(
-        menus=('global', 'footer'),
-        views=(
-            xcbv.ModelView.factory(
-                model=YourModel,
-                # updates views.list, views.update
-                children=xcbv.ModelView.views.update(
-                    YourListView.factory(
-                        # by default only staff sees new views, this opens for all
-                        allows=lambda self: True
-                    ),
-                    YourUpdateView.factory(
-                        # could be set here or inside the view
-                        path='specialupdate/<slug0>/<slug1>',
-                    ),
-                    YourDetailView.factory(
-                        # just adding a sub object list view inside URL and
-                        # permission tree like a Poney, is this going to be a
-                        # list of child models from the above ModelView model ?
-                        views=xcbv.ModelListView(
-                            model=YourChildModel,
-                        )
-                    )
-                ),
-                icon="fa-love",
+    urlpatterns = xcbv.Router(
+        xcbv.TemplateView.factory(
+            name='index', # default /index to index.html
+        ),
+        YourOtherView,
+        xcbv.Router(
+            YourCreateView(url='/absolute/url'),
+            YourListView(
+                # by default only staff sees new views, this opens for all
+                allows=lambda self: True
+            ),
+            YourUpdateView(
+                # could be set here or inside the view
+                path='specialupdate/<slug0>/<slug1>',
+            ),
+            ObjectFormView(
+                form_class=YourCustomForm
+                key='custom',
+            ),
+            YourDetailView(
+                # just adding a sub object list view inside URL and
+                # permission tree like a Poney, is this going to be a
+                # list of child models from the above ModelView model ?
+                views=xcbv.ListView(
+                    model=YourChildModel,
```

Comment:
I think that's where I left it off. Turns out this is not necessary, for this kind of use cases just add list views to the router and set them with menus = ['details'] in crudlfap